test/src/main/java/mixins/InsertTestMixin.java

```diff
@@ -24,9 +24,11 @@ abstract class InsertTestMixin {
 abstract class Inner2Mixin {
     /**
      * Soo... this returns stuff from the {@code integer} of type {@link AtomicInteger}.
-     * @param integer the param
+     * @param integer the param with very much text that goes on and on for a very long time because the goal is to hopefully trick it into forcing a
+     *                multiline tag
+     * @param replacement and now we're going to see how it handles multiple parameters
```

Comment:
For the record, this is the output of `applyDocsSpoon`:
```java
         /**
          * Soo... this returns stuff from the {@code integer} of type {@link java.util.concurrent.atomic.AtomicInteger}.
          * @param integer     the param with very much text that goes on and on for a very
          *                    long time because the goal is to hopefully trick it into
          *                    forcing a multiline tag
          * @param replacement and now we're going to see how it handles multiple parameters
          * @return the returned magic thingy
          */
         int weirdNr2(AtomicInteger integer, int replacement) {
            return integer.getAndSet(replacement);
         }
```